src/utils/sources-tree/addToTree.js

```diff
@@ -150,10 +160,18 @@ export function addToTree(
   const url = getURL(source.get("url"), debuggeeUrl);
   const debuggeeHost = getDomain(debuggeeUrl);
 
-  if (isInvalidUrl(url, source)) {
+  if (isInvalidUrl(url, source) || !isUnderRoot(url, projectRoot)) {
     return;
   }
 
-  const finalNode = traverseTree(url, tree, debuggeeHost);
+  const finalNode = traverseTree(url, tree, debuggeeHost, projectRoot);
   finalNode.contents = addSourceToNode(finalNode, url, source);
 }
+
+function isUnderRoot(url, projectRoot) {
+  if (!projectRoot) {
```

Comment:
Not sure I'd expect isUnderRoot to return true if there is no root.

Reply:
It's a hack when the project root is not set :P